frontend container doesn't compile anymore

Hi,

just tried to start workadventu.re and got:
System is ubuntu 18.04, doesn't have a hostname only IP tried to use single-domain.yaml

```
front_1          | ✖ ｢wdm｣: Hash: 44fd65452653348e1b25
front_1          | Version: webpack 4.44.2
front_1          | Time: 46602ms
front_1          | Built at: 04/27/2021 12:27:22 AM
front_1          |                           Asset      Size      Chunks                         Chunk Names
front_1          |                   iframe_api.js  1.49 MiB  iframe_api  [emitted]              iframe_api
front_1          |                      index.html  1.74 KiB              [emitted]
front_1          | js/main.c1d6a3f46dd6a6788984.js  24.6 MiB        main  [emitted] [immutable]  main
front_1          |  style.49237e278b5798517035.css  74.4 KiB        main  [emitted] [immutable]  main
front_1          | Entrypoint main = style.49237e278b5798517035.css js/main.c1d6a3f46dd6a6788984.js
front_1          | Entrypoint iframe_api = iframe_api.js
front_1          |  [0] multi (webpack)-dev-server/client?http://0.0.0.0:8080&sockPort=80 ./src/index.ts 40 bytes {main} [built]
front_1          | [12] multi (webpack)-dev-server/client?http://0.0.0.0:8080&sockPort=80 ./src/iframe_api.ts 40 bytes {iframe_api} [built]
front_1          |  [./dist/resources/style/index.scss] 50 bytes {main} [built]
front_1          |  [./node_modules/phaser/dist/phaser.js] 5.99 MiB {main} [built]
front_1          |  [./node_modules/rxjs/_esm5/index.js] 3.33 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/index.js?http://0.0.0.0:8080&sockPort=80] (webpack)-dev-server/client?http://0.0.0.0:8080&sockPort=80 4.29 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.51 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.53 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/utils/createSocketUrl.js] (webpack)-dev-server/client/utils/createSocketUrl.js 2.91 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/utils/log.js] (webpack)-dev-server/client/utils/log.js 964 bytes {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/utils/reloadApp.js] (webpack)-dev-server/client/utils/reloadApp.js 1.59 KiB {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/client/utils/sendMessage.js] (webpack)-dev-server/client/utils/sendMessage.js 402 bytes {main} {iframe_api} [built]
front_1          |  [./node_modules/webpack-dev-server/node_modules/strip-ansi/index.js] (webpack)-dev-server/node_modules/strip-ansi/index.js 161 bytes {main} {iframe_api} [built]
front_1          |  [./src/iframe_api.ts] 5.34 KiB {iframe_api} [built]
front_1          |  [./src/index.ts] 4.8 KiB {main} [built]
front_1          |     + 500 hidden modules
front_1          |
front_1          | ERROR in   Error: Child compilation failed:
front_1          |   Entry module not found: Error: Can't resolve '/usr/src/app/dist/index.tmpl.htm  l.tmp' in '/usr/src/app':
front_1          |   Error: Can't resolve '/usr/src/app/dist/index.tmpl.html.tmp' in '/usr/src/app'  EntryModuleNotFoundError: Entry module not found: Error: Can't resolve '/usr/s  rc/app/dist/index.tmpl.html.tmp' in '/usr/src/app'
front_1          |
front_1          |   - Compilation.js:1075
front_1          |     [app]/[webpack]/lib/Compilation.js:1075:31
front_1          |
front_1          |   - NormalModuleFactory.js:401
front_1          |     [app]/[webpack]/lib/NormalModuleFactory.js:401:22
front_1          |
front_1          |   - NormalModuleFactory.js:130
front_1          |     [app]/[webpack]/lib/NormalModuleFactory.js:130:21
front_1          |
front_1          |   - NormalModuleFactory.js:224
front_1          |     [app]/[webpack]/lib/NormalModuleFactory.js:224:22
front_1          |
front_1          |   - async.js:2830
front_1          |     [app]/[neo-async]/async.js:2830:7
front_1          |
front_1          |   - async.js:6877
front_1          |     [app]/[neo-async]/async.js:6877:13
front_1          |
front_1          |   - NormalModuleFactory.js:214
front_1          |     [app]/[webpack]/lib/NormalModuleFactory.js:214:25
front_1          |
front_1          |   - Resolver.js:213
front_1          |     [app]/[enhanced-resolve]/lib/Resolver.js:213:14
front_1          |
front_1          |   - Resolver.js:285
front_1          |     [app]/[enhanced-resolve]/lib/Resolver.js:285:5
front_1          |
front_1          |
front_1          |   - child-compiler.js:131
front_1          |     [app]/[html-webpack-plugin]/lib/child-compiler.js:131:18
front_1          |
front_1          |   - Compiler.js:343
front_1          |     [app]/[webpack]/lib/Compiler.js:343:11
front_1          |
front_1          |   - Compiler.js:681
front_1          |     [app]/[webpack]/lib/Compiler.js:681:15
front_1          |
front_1          |
front_1          |   - Hook.js:154 AsyncSeriesHook.lazyCompileHook
front_1          |     [app]/[tapable]/lib/Hook.js:154:20
front_1          |
front_1          |   - Compiler.js:678
front_1          |     [app]/[webpack]/lib/Compiler.js:678:31
front_1          |
front_1          |
front_1          |   - Hook.js:154 AsyncSeriesHook.lazyCompileHook
front_1          |     [app]/[tapable]/lib/Hook.js:154:20
front_1          |
front_1          |   - Compilation.js:1423
front_1          |     [app]/[webpack]/lib/Compilation.js:1423:35
front_1          |
front_1          |
front_1          |
front_1          | Child HtmlWebpackCompiler:
front_1          |
front_1          |     ERROR in Entry module not found: Error: Can't resolve '/usr/src/app/dist/index.tmpl.html.tmp' in '/usr/src/app'
front_1          | Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js?url=false!node_modules/sass-loader/dist/cjs.js!dist/resources/style/index.scss:
front_1          |     Entrypoint mini-css-extract-plugin = *
front_1          |     [./node_modules/css-loader/dist/cjs.js?url=false!./dist/resources/style/style.css] 56.5 KiB {mini-css-extract-plugin} [built]
front_1          |     [./node_modules/css-loader/dist/cjs.js?url=false!./node_modules/sass-loader/dist/cjs.js!./dist/resources/style/index.scss] 8.28 KiB {mini-css-extract-plugin} [built]
front_1          |     [./node_modules/css-loader/dist/runtime/api.js] 1.57 KiB {mini-css-extract-plugin} [built]
front_1          |     [./node_modules/css-loader/dist/runtime/cssWithMappingToString.js] 2.21 KiB {mini-css-extract-plugin} [built]
front_1          | ℹ ｢wdm｣: Failed to compile.

```
edit: added code tags

also pusher container has a problem:

```
messages_1       | $ rm -rf ../back/src/Messages/generated && cp -rf generated/ ../back/src/Messages/generated
messages_1       | $ rm -rf ../front/src/Messages/generated && cp -rf generated/ ../front/src/Messages/generated
pusher_1         | Compilation error in /usr/src/app/src/Controller/IoSocketController.ts
pusher_1         | [ERROR] 00:57:35 ⨯ Unable to compile TypeScript:
pusher_1         | src/Controller/IoSocketController.ts(16,8): error TS2307: Cannot find module '../Messages/generated/messages_pb' or its corresponding type declarations.
pusher_1         | src/Controller/IoSocketController.ts(17,32): error TS2307: Cannot find module '../Messages/generated/messages_pb' or its corresponding type declarations.
pusher_1         |
messages_1       | $ rm -rf ../pusher/src/Messages/generated && cp -rf generated/ ../pusher/src/Messages/generated

```

--- a/docker-compose.single-domain.yaml
+++ b/docker-compose.single-domain.yaml
@@ -30,7 +30,8 @@ services:
       UPLOADER_URL: /uploader
       ADMIN_URL: /admin
       MAPS_URL: /maps
-      STARTUP_COMMAND_1: yarn install
+      STARTUP_COMMAND_1: ./templater.sh
+      STARTUP_COMMAND_2: yarn install
       TURN_SERVER: "turn:localhost:3478,turns:localhost:5349"
       # Use TURN_USER/TURN_PASSWORD if your Coturn server is secured via hard coded credentials.
       # Advice: you should instead use Coturn REST API along the TURN_STATIC_AUTH_SECRET in the Back container

